README.md

```diff
@@ -1,2 +1,16 @@
-# docs
-Public documentation
+# Write code for the Internet Computer
+
+You can get started right away by downloading the DFINITY Canister Software Development Kit (SDK) and cloning an existing sample project or by exploring the documentation. Check out the tiles below to begin building open services and software to run on the Internet Computer.
+
+- [*Get the SDK*](https://sdk.dfinity.org/docs/download.html) Download the full SDK to write code that runs directly on the internet.
+- [*Quick Start*](https://sdk.dfinity.org/docs/quickstart/quickstart.html) Deploy your first code to the Internet Computer in minutes.
+- [*Get Samples*](https://sdk.dfinity.org/docs/developers-guide/sample-apps.html) Access sample code, applications, and microservices.
+- [*Developer Tools*](https://sdk.dfinity.org/docs/developers-guide/introduction-key-concepts.html) Take a deeper dive into how you can use the SDK to create applications.
+- [*Language Guide*](https://sdk.dfinity.org/docs/language-guide/motoko.html) Explore Motoko, the  programming language for the Internet Computer.
+- [*Basic Tutorials*](https://sdk.dfinity.org/docs/developers-guide/tutorials-intro.html) Learn the basics with simple examples and step-by-step instructions for first-timers.
+
+# Contributing to the documentation
+
+The Internet Computer is a new technology stack that is unhackable, fast, scales to billions of users around the world, and supports a new kind of autonomous software that promises to reverse Big Tech’s monopolization of the internet. It allows developers to take on the monopolization of the internet, and return the internet back to its free and open roots. We're committed to connecting those who believe the same through our events, content, and discussions.
```

Comment:
This language is taken straight from the website.

Reply:
To reduce some of the "monopolization," here's an alternative also from some website copy:

It allows developers to create open, autonomous software and services and restores the Internet to its permissionless, innovative, and collaborative roots.